### DISCUSSION TITLE
[Kavita] Use the peexisting cover file for titles if exists, of using a generated one

### DISCUSSION THREAD

**Discussion by theAlevan** [Kareadita/Kavita#3086] [Ideas]
### Idea Description

Currently, for each title, Kavita generates a cover file from the media files. The title cover is the first cover generated for the first chapter.

But most downloaders place the official cover for the title in the root of its folder, usually cover.png. 

If exists, Kavita should use this file as title cover, instead of a generated one, if exists. 

For implementation, my ideea is, that if a file named "cover" exists (of course case insensitive), with an image file format supported by kavita (jpeg, png, etc), in the directory of a manga, comic or a book, then it should be by default the cover for a title.

### Idea Category

User Experience

### Duration of Using Kavita

around 1 year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Kalidibus** [comment]
Came here to submit this same request.
